### ISSUE TITLE
Question about loss weight

### ISSUE THREAD

**Issue by Ericzhang1** [aqlaboratory/openfold#23]
Hi all! Firstly, thanks for your work and effort! I noticed that in the config file, the weight for each loss is different than that in af2's paper. For example, the weight for angle loss is 1, instead of 0.3. Some of them, such as the violation loss, experimentally solved loss, have a weight of 0. Is there any reason that the weight is set up this way? For instance, for losses that have been assigned a weight of 0 in the implementation, are they still under testing? Thanks!

**Comment by gahdritz** [comment]
The default config contains the settings used for the first phase of AlphaFold training. Violation loss and experimentally resolved loss were only activated during the second fine-tuning phase. Both have been tested and are fully usable, though.

As for the angle loss: 0.3 is the default weight for distogram loss in paper, not FAPE loss, no?

**Comment by Ericzhang1** [comment]
> The default config contains the settings used for the first phase of AlphaFold training. DeepMind only activated violation loss and experimentally resolved loss during the fine-tuning phase. Both losses have been tested and are fully usable, though.
> 
> As for the angle loss: 0.3 is the default weight for distogram loss in the paper, not the torsion angle loss, no?

Thanks, it is much clear now. Btw, the angle loss is part of the auxiliary loss, which has a weight of 0.5.
